ironfish/src/metrics/meter.ts

```diff
@@ -13,27 +13,36 @@ import { RollingAverage } from './rollingAverage'
  * completd each tick cycle and record that in various
  * rolling averages.
  *
- * @TODO: Move RollingAverages to exponentially-weighted moving average (EWMA)
  * */
 export class Meter {
   private _started = false
-  private _rate1s: RollingAverage
-  private _rate5s: RollingAverage
-  private _rate1m: RollingAverage
-  private _rate5m: RollingAverage
-  private _average: RollingAverage
+  private _rate1s: EwmaAverage
+  private _rate5s: EwmaAverage
+  private _rate1m: EwmaAverage
+  private _rate5m: EwmaAverage
+  private _average: EwmaAverage
```

Comment:
this one is not a time series value, but a mathematical average, so should it be EwmaAverage, or RollingAverage?